source/documentation/deploying_services/route_services.md

```diff
@@ -139,6 +139,15 @@ This example app uses the `X-Forwarded-For` header added by the GOV.UK PaaS rout
 Using an IP address to authenticate HTTP requests is not sufficient as a standalone authentication mechanism.
 You should use other authentication and authorization methods in your apps.
 
+Note: the example nginx configuration used in this app will not work out of the box
+when used with requests routed through a CDN. This includes requests routed through our
+own AWS CloudFront-based [cdn-route services](/deploying_services/use_a_custom_domain/). For these to work, [`set_real_ip_from`](https://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from)
+entries need to be added to `nginx.conf` for all IP ranges the CDN may forward
```

Comment:
```suggestion
entries to `nginx.conf` for all IP ranges the CDN may forward
```

Reply:
Could we avoid the passive voice here by asking the user directly to carry out this action?